engines/python/setup/djl_python/input_parser.py

```diff
@@ -78,22 +78,27 @@ def parse_input_with_formatter(inputs: Input, **kwargs) -> ParsedInput:
     input_formatter_function = configs.input_formatter if configs.input_formatter else format_input
     for i in range(start_batch_id, len(batch)):
         input_item = batch[i]
+        request_id = input_item.get_property("requestId")
```

Comment:
So, will the customers give us the request_id?

Reply:
Yes, there are details in https://github.com/deepjavalibrary/djl-serving/pull/2363. We expose a property `request_id_header_key` that allows users to specify which header to use as the requestId. If not provided, we default to `X-Amzn-SageMaker-Inference-Id` on SageMaker, or `x-request-id` otherwise